schemas/observer.yml

```diff
@@ -31,6 +31,18 @@
       type: keyword
       description: >
         Hostname of the observer.
+    - name: name
+      level: core
```

Comment:
can we make these fields extended for now? Easy to move it to core later but not sure about the other way around.

Reply:
👍  for `level: extended` for these new additions